src/roslibpy/actionlib.py

```diff
@@ -465,11 +481,23 @@ def set_preempted(self):
         LOGGER.info("Action server {} preempting current goal".format(self.server_name))
 
         with self._lock:
+
+            self.status_message["status_list"] = [
```

Comment:
Tiny, nitpicky comment ;)
Since the status portion of both the Status and the Result messages is the same, would it not be cleaner to just create it once? (same in `set_succeeded`)

Reply:
Fixed this and added tests. I'll merge.